views/event_details.hbs

```diff
@@ -1,22 +1,23 @@
 <div class="eventDetailsHeader">
     {{>header}}
-  <div class="prevBtn ">
-    <a href="{{{back}}}"><span class="glyphicon glyphicon-chevron-left"></span></a>
-  </div>
 </div>
 
-<div class="event-details text-center">
-  <h1 class="text-warning eventName">{{event.en.name}}</h1>
-  {{#if imageUrl}}
-  <img src="{{event.imageUrl}}" alt="event picture">
+<div class="event-details">
+  {{#if event.imageUrl}}
+  <img class="event-img center" src="{{event.imageUrl}}" alt="event picture">
```

Comment:
Needs indenting. Also, I think it would be easier to use just a `div` and set the `background-image` here. Then we can give it a `background-position: center`, `background-size: cover` and limit the height of the image - at the moment as you scale the page the image scales too.